client/client-multi/client-c/build.sh

```diff
@@ -38,7 +38,7 @@ COLLECT_COVERAGE=0
 prepare_build() {
     mkdir -p build;
     cd build;
-    cmake -DKAA_DEBUG_ENABLED=$DEBUG_ENABLED -DKAA_MAX_LOG_LEVEL=$MAX_LOG_LEVEL -DKAA_UNITTESTS_COMPILE=$UNITTESTS_COMPILE -DKAA_COLLECT_COVERAGE=$COLLECT_COVERAGE ..;
+    cmake -DKAA_DEBUG_ENABLED=$DEBUG_ENABLED -DKAA_MAX_LOG_LEVEL=$MAX_LOG_LEVEL -DKAA_UNITTESTS_COMPILE=$UNITTESTS_COMPILE -DKAA_COLLECT_COVERAGE=$COLLECT_COVERAGE .. -DCMAKE_C_FLAGS="-Werror";
```

Comment:
Not critical comment: let's think about passing build type via CMAKE_BUILD_TYPE . Debug build may expose then  the -Werror flag somewhere in listfiles using CMAKE_C_FLAGS_DEBUG. This adds some indirection and ability to add more flags if needed for development purposes.

ATM this not critical, just a cause to have a chat.